.github/workflows/dsf_poc_standalone.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     uses: imperva/dsfkit/.github/workflows/dsf_poc_cli.yml@master
```

Comment:
just pay attention that the yml itself is always from master, but I guess it is OK for us since it is not commonly changed